src/allmydata/immutable/upload.py

```diff
@@ -1064,11 +1063,11 @@ def _hash_and_encrypt_plaintext(self, data, hash_only):
             self._plaintext_hasher.update(chunk)
             self._update_segment_hash(chunk)
             # TODO: we have to encrypt the data (even if hash_only==True)
-            # because pycryptopp's AES-CTR implementation doesn't offer a
-            # way to change the counter value. Once pycryptopp acquires
+            # because the AES-CTR implementation doesn't offer a
+            # way to change the counter value. Once it acquires
             # this ability, change this to simply update the counter
-            # before each call to (hash_only==False) _encryptor.process()
```

Comment:
did we verify this comment is still true?

Reply:
Insofar as "I can't see a way to set the counter in cryptography, either" without doing "private stuff" then yes.